README.md

```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+Alternatively, you can also use the fully-featured official [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
     pip3 install tldr
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+Then you have direct access to less complicated, easier-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`
 
 However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+out the [PDF version](https://tldr.sh/assets/tldr-book.pdf),
+which is always up-to-date with the most recent pages and commits.
```

Comment:
Maybe we should mention the web clients which can also be used without installing anything (and are arguably more convenient than a huge document covering all pages). The PDF is likely a better choice for offline consumption, so we might want to highlight that aspect instead.

Reply:
Agreed, I will make a full suggestion tomorrow.

Reply:
```suggestion
which is always up-to-date with the latest pages and commits.
```

* "most recent" might be a bit wordy in this case, thoughts?

Reply:
@SethFalco your suggestion here has been applied.